cmd/ceremony/main.go

```diff
@@ -794,6 +794,11 @@ func main() {
 	var ct struct {
 		CeremonyType string `yaml:"ceremony-type"`
 	}
+
+	// We are intentionally using non-strict unmarshaling to read the top level
+	// tags to populate the "ct" struct for use in the switch statement below.
+	// Further strict processing of each yaml node is done on a case by case basis
+	// inside the switch statement.
```

Comment:
The strict unmarshalling inside the switch statement uses its own mechanism, rather than relying on your new shiny strictyaml package:
https://github.com/letsencrypt/boulder/blob/fdfea0d469b61a915806943db5c1bddd0d417eb7/cmd/ceremony/main.go#L467-L472

Is there a reason to leave that as-is, rather than fully unifying it as
```go
func rootCeremony(configBytes []byte) error {
	var config rootConfig
	err := strictyaml.Unmarshal(configBytes, &config)
```